config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -592,3 +592,35 @@ periodics:
     testgrid-alert-email: kubernetes-secrets-store-csi-driver@googlegroups.com
     description: "Run vulnerability scans for Secrets Store CSI driver images."
     testgrid-num-columns-recent: '30'
+
+- interval: 24h
+  name: periodic-secrets-store-csi-driver-upgrade-test-azure
+  decorate: true
+  decoration_config:
+    timeout: 30m
+  labels:
```

Comment:
These are the required labels

https://github.com/kubernetes/test-infra/blob/f668b08714d2974ff1fbe68302050d74e4bbedf4/config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml#L140-L146